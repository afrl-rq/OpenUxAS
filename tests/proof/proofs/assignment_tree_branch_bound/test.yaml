filenames: ["assignment_tree_branch_bound.ads", "algebra.ads"]
level: 4
