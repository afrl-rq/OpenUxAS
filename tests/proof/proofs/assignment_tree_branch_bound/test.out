assignment_tree_branch_bound.adb:271:08: medium: "All_Actions_In_Map" is recursive, terminating annotation could be incorrect
assignment_tree_branch_bound.adb:437:19: medium: assertion might fail
assignment_tree_branch_bound.adb:469:22: medium: "Prove_TaskOptionId_In_Map" is recursive, terminating annotation could be incorrect
assignment_tree_branch_bound.adb:1071:15: medium: precondition might fail, cannot prove Has_Key (Container, Key)
assignment_tree_branch_bound.adb:1294:69: medium: precondition might fail, cannot prove Has_Key (Container, Key)
assignment_tree_branch_bound.adb:1301:17: medium: precondition might fail, cannot prove Has_Key (Container, Key)
assignment_tree_branch_bound.adb:1352:25: medium: "Check_Actions_In_Map_Rec" is recursive, terminating annotation could be incorrect
assignment_tree_branch_bound.adb:1571:19: medium: loop invariant might fail in first iteration, cannot prove Contains_Corresponding_TaskOption (Result.Assignment_Sequence, Element (Result.Vehicle_Assignments, Int64_VehicleAssignmentCost_Maps_K.Get (Keys (Result.Vehicle_Assignments), K)).Last_TaskOption)
assignment_tree_branch_bound.adb:1919:07: warning: statement has no effect
assignment_tree_branch_bound.adb:1919:18: warning: "Algebra" is set by "Free_Tree" but not used after the call
algebra.ads:59:08: medium: "Is_Present" is recursive, terminating annotation could be incorrect
